social/src/main/java/bisq/social/user/profile/UserProfileService.java

```diff
@@ -287,6 +288,11 @@ public Observable<UserProfile> getSelectedUserProfile() {
     public ObservableSet<UserProfile> getUserProfiles() {
         return persistableStore.getUserProfiles();
     }
+    
+    public Collection<ChatUser> getMentionableChatUsers() {
+        // TODO: implement logic
+        return getUserProfiles().stream().map(UserProfile::getChatUser).toList();
```

Comment:
> The usernames are rerived from the list of past interactions with users (private channels) and the users in the chat history

I didn't yet implemented this logic - returning all users for now. Let me know if it's important to have it now or it's okay to finish it in some further pull request. I guess this logic will be more advanced, depending on which channel we are etc.

Similar situation with [`getMentionableChannels`](https://github.com/bisq-network/bisq2/pull/288/files#diff-c9a1cb827d7fabf810d2fb3696f18c945c065230e6909e43152c9737fef33c65R417).

Reply:
We can do that later...